script/integration.go

```diff
@@ -196,7 +196,7 @@ func setBash() {
 		os.Exit(1)
 	}
 
-	bashPath = strings.TrimSpace(string(out))
+	bashPath = strings.TrimSpace(strings.Split(string(out), "\n")[0])
```

Comment:
Calling `strings.Split()` on an empty string returns an empty `[]string`, so accessing `[0]` will cause go to panic. That's probably a super rare scenario, but I think it's worth adding a check right above this line:

```go
if len(out) == 0 {
    fmt.Printf("No output from '%s bash'\n", findcmd)
    os.Exit(1)
}